css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  padding: 0;
+  background-color: #05a8f3;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Lato", sans-serif;
+  margin: 20px 40px;
+  padding-top: 10px;
+  padding-bottom: 15px;
+  text-align: center;
+}
+
+.heading {
+  font-size: 40px;
+  color: white;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 80px;
+  color: white;
+}
+
+.images-column {
+  width: 25%;
+  display: block;
+  background-color: #05a8f3;
+  padding: 15px 20px;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  border-radius: 50px;
+}
+
+.img-container {
+  width: auto;
+  height: 260px;
+  margin: 0 auto;
+  border-radius: 50%;
+  object-fit: fill;
+  background-color: white;
+}
+
+.resize-img {
+  width: 75%;
+  height: 260px;
+  border-radius: 50%;
```

Comment:
Try putting the border radius on the div holding the image instead of the image itself.

Reply:
Updated